AbSuppliers.php

```diff
@@ -287,6 +287,42 @@ public function displaySupplierPartners($atts)
         return $response;
     }
 
+    public function displaySupplierPartnersForEnergy($atts)
+    {
+        $atts['partners_only'] = true;
+
+        $atts = $this->processMultipleProductCats($atts);
+
+        list($atts, $supplierLogos) = $this->preparedSuppliersLogoData($atts);
+
+        $breakPoint = $atts['mod'];
+        $counter    = 0;
+        $response   = '<div class="row">';
+
+        foreach ($supplierLogos as $supplier) {
+            $atts['link'] = $this->generateProviderLink($atts, $supplier);
+
+            // If $counter is divisible by $breakpoint mod, create new row
+            if ($counter % $breakPoint == 0 && $counter != 0) {
+                // New div row
+                $response .= '</div><div class="row">';
+            }
+            $response .= '<' . $atts['mark-up'] .
```

Comment:
Ik weet niet precies wat processMultipleProductCats of preparedSuppliersLogoData doet met atts, maar als je niet zeker bent dat al deze dingen geset zijn ('mark-up' en 'mark-up-class' enzo), dan kun je beter boven een array met defaults maken (of hier isset gebruiken). Error reporting staat overal uit, maar dat betekent niet dat je ze niet moet voorkomen :)